### ISSUE TITLE
Version 2.1 does not work

### ISSUE THREAD

**Issue by lcottret** [draeger-lab/ModelPolisher#128]
Hi, 

I would like to add pathway information in the polished models. The issue #97 says that this feature is implemented in the branch 2.1.
However, when I  run "docker compose up" in this branch, the following error appears : 
```
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "--config-file": executable file not found in $PATH: unknown
```

OS : ubuntu 22.06
Docker version : 27.1.2

Regards,
Ludo

**Comment by Schmoho** [comment]
Hi,

sorry for the late reply, I somehow missed this ticket.
First, please be aware that the 2.1 branch is very much not stable and is not intended to be stable, as it is a development branch.

I am missing some information here: I am not certain which state of the `docker-compose.yml` you are referring to, there have been quite some changes on them lately. In particular, I would need to know which polisher image you were using.

I am aiming to finish the 2.1 release by October. Until then, I would advise you to use [this release](https://github.com/draeger-lab/ModelPolisher/releases/tag/2.1-beta), or try the experimental API currently supported by the [MPClient](https://github.com/draeger-lab/MPClient).

I am aware that the Polisher is at the moment somewhat inconvenient to use. Going forward (i.e. with the stable 2.1 release) the preferred mode of usage will be the Python HTTP Client.

**Comment by lcottret** [comment]
Thanks ! Don't worry, I'm going to wait for the 2.1 release and look at the MPClient.

**Comment by Schmoho** [comment]
Hello @lcottret

Sorry for the unexpectedly long time to respond. The https://github.com/draeger-lab/MPClient project is now available on PyPI test and the 2.1 version has been released and is considered stable for now. In case you are still interested in using the software and encounter further issues, please don't hesitate to open an issue here.

I will however close this for now.
